src/renderer/i18n/locales/it-IT.js

```diff
@@ -0,0 +1,964 @@
+    SELECT_OPTION: 'Seleziona un\'opzione',
+    DELEGATE: 'Delegato',
+    WALLETS: 'Portafogli',
+    PERIOD: 'Periodo'
+  },
+
+  CONTACT_REMOVAL_CONFIRMATION: {
+    QUESTION: 'Sei sicuro di voler rimuovere questo contatto?'
+  },
+
+  ENCRYPTION: {
+    DECRYPTING: 'Decifrando il portafoglio con password...',
+    ENCRYPTING: 'Cifrando il portafoglio con password...',
+    FAILED_DECRYPT: 'Fallito nel decifrare la frase segreta',
+    FAILED_ENCRYPT: 'Fallito nel cifrare la frase segreta'
+  },
+
+  PEER: {
+    BEST: 'Connetti al migliore',
+    CONNECTED: 'Connesso al peer',
+    CONNECT_CUSTOM: 'Connetti peer personalizzato',
+    CONNECT_FAILED: 'Impossibile connettersi al peer',
+    DELAY: 'Ritardo',
+    DISCONNECT: 'Disconnesso dal peer',
+    FAILED_REFRESH: 'Impossibile aggiornare i peers',
+    HEIGHT: 'Altezza Blocco',
+    LAST_CHECKED: 'Ultimo controllo',
+    NONE: 'Nessuno',
+    NO_CONNECT: 'Impossibile connettersi',
+    PEER: 'Peer:',
+    STATUS_CHECK_FAILED: 'Status check fallito',
+    WRONG_NETWORK: 'Rete errata'
+  },
+
+  VALIDATION: {
+    TOO_LONG: '\'{0}\' è troppo lungo',
+    INVALID_URI: 'URI non valido',
+    INVALID_FORMAT: 'Formato non valido',
+    MAX_LENGTH: 'Max {0}',
+    MUST_BE_GREATER_THAN: 'Deve essere maggiore di {0}',
+    NOT_MATCH: '\'{0}\' non coincide con \'{1}\'',
+    NOT_VALID: '\'{0}\' non è valido',
+    NOT_NUMERIC: '\'{0}\' non è numerico',
+    NO_SCHEME: '\'{0}\' non contiene \'http://\' o \'https://\'',
+    NAME: {
+      DUPLICATED: 'Il nome \'{0}\' esiste già',
+      EXISTS_AS_CONTACT: 'Il nome \'{0}\' è già stato assegnato a un contatto',
+      EXISTS_AS_WALLET: 'Il nome \'{0}\'è già stato assegnato a un portafoglio',
+      MAX_LENGTH: 'Il nome deve avere meno di {0} caratteri.',
+      MIN_LENGTH: 'Il nome deve avere almeno 1 carattere | Il nome deve avere almeno {n} caratteri.'
+    },
+    PASSWORD: {
+      TOO_SHORT: 'La tua password deve essere lunga lameno {0} caratteri',
+      NUMBERS: 'La tua password deve contenere almeno 1 numero',
+      SPECIAL_CHARACTERS: 'La tua password deve contenere almeno 1 carattere speciale',
+      NO_MATCH: 'La tua password non coincide'
+    },
+    ADDRESS: {
+      EXISTS_AS_CONTACT: 'L\'indirizzo \'{0}\' è già stato aggiunto come contatto',
+      EXISTS_AS_WALLET: 'L\'indirizzo \'{0}\'è già stato importato come portafoglio'
+    },
+    PUBLIC_KEY: {
+      INVALID_LENGTH: 'La chiave pubblica deve avere una lunghezza di 66 caratteri'
+    },
+    VENDOR_FIELD: {
+      LIMIT_REACHED: 'È possibile inserire un massimo di {0} caratteri'
+    },
+    REQUIRED: 'The \'{0}\' is required',
+    SEND_NOT_ENABLED: 'L\'invio non è abilitato per il portafoglio selezionato',
+    WALLET_NOT_ACTIVE: 'Seleziona un portafoglio a apri l\'URI nuovamente',
+    WARNING_BIP39: '\'{0}\' è una frase segreta BIP39',
+    WARNING_NOT_BIP39: '\'{0}\' non è una frase segreta BIP39'
+  },
+
+  LANGUAGES: {
+    'en-US': 'English',

```

Comment:
Each language in Italian, not in English.

Reply:
Ok, but shouldn't be any language name displayed in the original language?
Example: I set the app language to Chinese accidentally. Now, when choosing another language I will see all languages dispalyed in Chinese. Ok there are flags, but let me know what you think about it.

Reply:
Yes, that's something that I've been thinking, and yes, probably it's betterto keep each language on their original form. In that case, change this file and `en-US` to use "Italiano" instead of "Italian".